voltixApp/VoltixApp/Views/Vault/VaultPairDetailView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  VaultPairDetailView.swift
+//  VoltixApp
+//
+//  Created by Enrique Souza Soares on 10/04/2024.
+//
+import Foundation
+import SwiftUI
+
+struct VaultPairDetailView: View {
+    let vault: Vault
+    
+    struct DeviceInfo {
```

Comment:
Can you please move it to a Model instead

Reply:
Done